pyproject.toml

```diff
@@ -51,9 +51,9 @@ warn_required_dynamic_aliases = true
 warn_untyped_fields = true
 
 [tool.ruff]
-ignore = ["E203", "E402", "E501"]
+lint.ignore = ["E203", "E402", "E501", "C901"]
```

Comment:
[C901](https://docs.astral.sh/ruff/rules/complex-structure/) highlights all functions with a McCabe complexity > n. Given how far darwin-py is into it's development, I don't think we should consider this. It might be worth highlighting for `darwin.future`